datafusion/core/tests/user_defined_plan.rs

```diff
@@ -551,11 +552,7 @@ fn accumulate_batch(
         .downcast_ref::<StringArray>()
         .expect("Column 0 is not customer_id");
 
-    let revenue = input_batch
-        .column(1)
-        .as_any()
-        .downcast_ref::<Int64Array>()
-        .expect("Column 1 is not revenue");
+    let revenue = as_int64_array(input_batch.column(1)).unwrap();
```

Comment:
I think that you can still keep `.expect("Column 1 is not revenue");`?

Reply:
I agree but I am not sure if we will give specific error to user. If it is the preferred way, I will change it

Reply:
As this is test code, I think if the test data is correct, it must return a Int64Array for column 1. `Column 1 is not revenue` as error message, I think its purpose is to indicate that.

Both (previous and this PR) are fine. I'd keep it but it looks okay too for this change.

Reply:
Yeah I agree with @viirya